functions/zoom-downloader.py

```diff
@@ -119,17 +127,55 @@ def zoom_series_id(self):
     def start_time(self):
         return self.data['start_time']
 
+    @property
+    def end_time(self):
+        return self.data['end_time']
+
+    @property
+    def duration(self):
+        return self.data['duration']
+
     @property
     def created(self):
         utc = datetime.strptime(
-            self.start_time, '%Y-%m-%dT%H:%M:%SZ') \
+            self.start_time, TIMESTAMP_FORMAT) \
             .replace(tzinfo=timezone('UTC'))
         return utc
 
+    @property
+    def shorter_than_minimum_duration(self):
+        """
+        Interpret duration of meeting and determine whether duration is
+        shorter than the MINIMUM_DURATION in minutes.
+        """
+
+        try:
+            if self.duration:
+                if len(self.duration) == len("00:00"):
```

Comment:
this is a a little weird but fine. i've also used a try/catch loop when i'm not sure what format is going to work on an incoming string.

Reply:
I don't like it so I rewrote it. Changed the duration @property function so that it standardizes the duration format. Now shorter_than_minimum_duration can be much simpler. Also added tests.